app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/dao/VpnAppTrackerBlockingDao.kt

```diff
@@ -136,6 +136,9 @@ interface VpnAppTrackerBlockingDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insertIntoManualAppExclusionList(excludedApp: AppTrackerManualExcludedApp)
 
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertIntoManualAppExclusionList(excludedApps: List<AppTrackerManualExcludedApp>)
```

Comment:
Added this because it's significantly faster to insert a list than do a number of inserts.